app/helpers/sessions_helper.rb

```diff
@@ -21,6 +21,10 @@ def current_user
     end
   end
 
+  def current_user? user
+    user && user == current_user
```

Comment:
```suggestion
    user && user != current_user
```
Như thế này chứ em nhỉ. Ở trên bỏ phủ định đi
Cách làm của em thì nếu trường hợp user = nill 
Thì !current_user?(user) => true mà